lib/ret_web/controllers/page_controller.ex

```diff
@@ -227,10 +268,17 @@ defmodule RetWeb.PageController do
     "text/html; charset=utf-8"
   end
 
-  defp render_chunks(conn, chunks, content_type) do
+  defp render_chunks(conn, chunks, content_type, include_newlines) do
+    resp =
+      if include_newlines do
```

Comment:
any reason we would ever want to not do this? seems more correct since iirc the chunks were split on newline (i'm surprised we didn't have bugs unless i'm misunderstanding this code)

Reply:
Yeah, maybe I can just add the newlines always. Will test it out to see if anything breaks.